package.json

```diff
@@ -4,7 +4,7 @@
   "description": "RPC library for AMQP protocol",
   "main": "index.js",
   "scripts": {
-    "test": "gulp test && gulp inspect-queues",
```

Comment:
Why was `&& gulp inspect-queues` removed?

@yamii, We're still using that, aren't we?